components/decohelp/pages/ui/ForwardBackButtons/ForwardBackButtons.tsx

```diff
@@ -1,58 +1,80 @@
-export interface ForwardBackContent {
-  previous?: {
-    href?: string;
-    title?: string;
-    category?: string;
-    previousLabel?: string;
-  };
-  next?: {
-    href?: string;
-    title?: string;
-    category?: string;
-    nextLabel?: string;
-  };
+import { useEffect, useState } from "preact/hooks";
+
+interface PageButton {
+  title: string;
+  href: string;
 }
 
-export default function ForwardBackButtons(
-  { previous, next }: ForwardBackContent,
-) {
+export default function ForwardBackButtons() {
   const upper = "text-neutral-900 text-sm font-bold leading-tight";
-  const category = "font-normal";
   const lower = "text-gray-900 font-medium";
+
+  const [previous, setPrevious] = useState<PageButton>({
+    title: "",
+    href: "",
+  });
+  const [next, setNext] = useState<PageButton>({
+    title: "",
+    href: "",
+  });
+
+  const isEnglish = window.navigator.language.startsWith("en");
+  const [isEnglishState, setIsEnglishState] = useState(isEnglish);
+
+  useEffect(() => {
+    const encodedURL = window.location.href;
```

Comment:
may cause some issues with query parameters and hashs.

```suggestion
    const encodedURL = window.location.href.split(window.location.search||window.location.hash||/[?#]/)[0];
```

Reply:
![image](https://github.com/deco-sites/starting/assets/76822093/90e6b489-ade5-4ad9-b7d3-f6fbffb411b5)
done